### PR TITLE
HEAD method to obtain whether an entity exist or not

### PR DESCRIPTION
**Pull request by Marx314**

### CODE REVIEW

**Comment by mbelang** [comment]
:+1:

File: almanach/adapters/api_route_v1.py [hunk 1]
```diff
@@ -277,6 +277,16 @@ def update_instance_entity(instance_id):
     return result
 
 
+@api.route("/entity/<entity_id>", methods=["HEAD"])
+@authenticated
+def entity_exist(entity_id):
```
Comment (fguillot) [inline almanach/adapters/api_route_v1.py]: `entity_exists()` with a `s`?

File: almanach/adapters/api_route_v1.py [hunk 2]
```diff
@@ -277,6 +277,16 @@ def update_instance_entity(instance_id):
     return result
 
 
+@api.route("/entity/<entity_id>", methods=["HEAD"])
+@authenticated
+def entity_exists(entity_id):
+    logging.info("Does entity with id %s exist", entity_id)
```
Comment (fguillot) [inline almanach/adapters/api_route_v1.py]: exists

File: almanach/adapters/api_route_v1.py [hunk 3]
```diff
@@ -277,6 +277,16 @@ def update_instance_entity(instance_id):
...
+    response = Response('', 404)
+    if controller.entity_exist(entity_id=entity_id):
```
Comment (fguillot) [inline almanach/adapters/api_route_v1.py]: exists

File: tests/core/test_controller.py [hunk 4]
```diff
@@ -723,6 +723,24 @@ def test_instance_rebuilded(self):
             "2015-10-21T16:25:00.000000Z"
         )
 
+    def test_entity_exist(self):
```
Comment (fguillot) [inline tests/core/test_controller.py]: exists

**Pull request merged**